src/scripts/site.js

```diff
@@ -202,61 +202,75 @@ var Dashboard = {
                         apiClient.enableAutomaticNetworking = !1, apiClient.manualAddressOnly = !0, connectionManager.addApiClient(apiClient), window.ApiClient = apiClient, localApiClient = apiClient, console.log("loaded ApiClient singleton"), resolve()
                     });
                     resolve()
-                })
-            })
-        })
+                });
+            });
+        });
     }
 
     function returnFirstDependency(obj) {
         return obj
     }
 
     function getSettingsBuilder(UserSettings, layoutManager, browser) {
-        return UserSettings.prototype.enableThemeVideos = function(val) {
-            return null != val ? this.set("enableThemeVideos", val.toString(), !1) : (val = this.get("enableThemeVideos", !1), val ? "false" !== val : !layoutManager.mobile && !browser.slow)
-        }, UserSettings
+        UserSettings.prototype.enableThemeVideos = function(val) {
+            return null != val
+                ? this.set("enableThemeVideos", val.toString(), !1)
+                : (val = this.get("enableThemeVideos", !1), val ? "false" !== val : !layoutManager.mobile && !browser.slow);
+        };
```

Comment:
maybe de-uglify further?
```suggestion
        UserSettings.prototype.enableThemeVideos = function(val) {
            if (val != null) {
                return this.set('enableThemeVideos', val.toString(), false);
            } else {
                val = this.get('enableThemeVideos', false);
                if (val !== 'false') {
                    return !layoutManager.mobile;
                } else {
                    return !browser.slow;
                }
            }
        };
```

Reply:
done, plus some extra in apphost